packages/fx-core/src/common/featureFlags.ts

```diff
@@ -44,6 +44,10 @@ export function isCopilotPluginEnabled(): boolean {
   return isFeatureFlagEnabled(FeatureFlagName.CopilotPlugin, false);
 }
 
+export function isMECopilotPluginEnabled(): boolean {
+  return isFeatureFlagEnabled(FeatureFlagName.MECopilotPlugin, false);
```

Comment:
We need to sync later about the feature flag. Thought we also need to make sure COPILOT_PLUGIN is true. We could work update that in the next PR.